scene/animation/animation_player.cpp

```diff
@@ -520,6 +520,7 @@ void AnimationPlayer::play_section(const StringName &p_name, double p_start_time
 	c.started = true;
 
 	_set_process(true); // Always process when starting an animation.
+	paused = false;
```

Comment:
```suggestion
```

I believe adding flags that aren't actually used within the core is undesirable.

In other words, since the core doesn't define what constitutes a pause using this flag, this would invert the dependency relationship between definition and behavior.